src/views/CountryOperations/AppealOperationTable/index.tsx

```diff
@@ -0,0 +1,183 @@
+import Container from '#components/Container';
+import {
+    useSortState,
+    SortContext,
+    getOrdering,
+} from '#components/Table/useSorting';
+import Pager from '#components/Pager';
+import Link from '#components/Link';
+import Table from '#components/Table';
+import {
+    createStringColumn,
+    createNumberColumn,
+    createDateColumn,
+    createLinkColumn,
+    createProgressColumn,
+} from '#components/Table/ColumnShortcuts';
+import {
+    type GoApiResponse,
+    useRequest,
+} from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type AppealTableItem = NonNullable<GoApiResponse<'/api/v2/appeal/'>['results']>[number];
+const keySelector = (appeal: AppealTableItem) => appeal.id;
+
+const now = new Date().toISOString();
+const PAGE_SIZE = 5;
+
+interface Props {
+    countryId?: string;
+    countryName?: string;
+}
+
+function AppealOperationTable(props: Props) {
+    const {
+        countryId,
+        countryName,
+    } = props;
+
+    const strings = useTranslation(i18n);
+
+    const sortState = useSortState();
+    const { sorting } = sortState;
+
+    const [page, setPage] = useState(1);
+    const viewAllOperations = resolveToComponent(
+        strings.allOperations,
+        {
+            name: countryName,
+        },
+    );
+    const {
+        pending: countryAppealPending,
+        response: countryAppealResponse,
+    } = useRequest({
+        skip: isNotDefined(countryId),
+        preserveResponse: true,
+        url: '/api/v2/appeal/',
+        query: {
+            end_date__gt: now,
+            limit: PAGE_SIZE,
+            offset: PAGE_SIZE * (page - 1),
+            ordering: getOrdering(sorting),
+            country: Number(countryId),
+        },
+    });
+    const {
+        emergency: emergencyRoute,
+        allAppeals: allAppealsRoute,
+    } = useContext(RouteContext);
+
+    const columns = useMemo(
+        () => ([
+            createDateColumn<AppealTableItem, string>(
+                'start_date',
+                strings.appealsTableStartDate,
+                (item) => item.start_date,
+                {
+                    sortable: true,
+                },
+            ),
+            createLinkColumn<AppealTableItem, string>(
+                'operation',
+                strings.appealsTableName,
+                (item) => item.name,
+                (item) => ({
+                    to: isDefined(item.id)
+                        ? generatePath(emergencyRoute.absolutePath, {
+                            emergencyId: item.id,
+                        })
+                        : undefined,
+                }),
+                {
+                    sortable: true,
+                },
+            ),
+            createNumberColumn<AppealTableItem, string>(
+                'event',
+                strings.appealsTableEmergency,
+                (item) => item.event,
+            ),
```

Comment:
This looks weird. What does "event" represent here?

Reply:
event represent emergencies